docs/kb/semgrep-cloud-platform/missing-pr-commments.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Use this reference to check why you may not be receiving Semgrep comments on PRs/MRs.
+tags:
+  - Semgrep CI
+  - PR Comments
+  - Rule Board
+---
+
+# Why am I not receiving PR/MR comments?
+
+If you have configured Semgrep in CI and Semgrep Cloud Platform to create comments when a rule generates a finding in a PR or MR, but you are not seeing those comments, check the following possibilities.
+
+## Are comments supported on your SCM?
+
+PR/MR Comments are currently supported for GitHub, GitLab, and Bitbucket. If you are using another SCM, comments are not yet supported. Bitbucket Data Center is also not supported yet.
+
+If you are using a self-hosted version of GitHub (GitHub Enterprise) or GitLab (GitLab Self-Managed), see [Integrating Semgrep Cloud Platform with self-hosted enterprise repositories](/docs/semgrep-cloud-platform/scm/) for more details on configuration.
+
+## Have you configured permissions and tokens correctly?
+
+### GitHub
+
+GitHub relies on the Semgrep GitHub app to make comments on code. To receive comments on a project, you must [onboard](/docs/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) the project to Semgrep Cloud Platform, or ensure that your GitHub app is configured with permissions for all repositories. See [Enabling GitHub pull request comments](/docs/semgrep-cloud-platform/github-pr-comments/) for details.
+
+### GitLab and Bitbucket
+
+For MR comments on GitLab and PR comments on Bitbucket, you must:
+
+* configure a token with appropriate permissions to comment on MRs or PRs
+* add the token as a variable in your project or workspace configuration
+* add the token value to your CI configuration
+
+The GitLab token should have `api` scope and be added to the project's CI/CD settings. See [Enabling GitLab merge request comments](/docs/semgrep-cloud-platform/gitlab-mr-comments/) for details.
```

Comment:
So - the tricky thing about combining the two (GitLab and Bitbucket) is that these 2 paragraphs, lines 33 and 34, are actually for 2 separate audiences. You may want to separate the sections. I'm sorry I couldn't help restructure this thing entirely, but if you want I can push a commit, just let me know.

Reply:
Is it okay to make two H4 sections here? I'm definitely open to that, but I did like highlighting that the outline is similar for both.

Reply:
That's what I went to for now, but we can definitely iterate later :)